internal/rbd/rbd_util.go

```diff
@@ -1924,25 +1927,41 @@ func genVolFromVolIDWithMigration(
 
 // setVolumeMetadata set PV/PVC/PVCNamespace metadata on RBD image.
 func (rv *rbdVolume) setVolumeMetadata(parameters map[string]string) error {
+	var err error
 	for k, v := range k8s.GetVolumeMetadata(parameters) {
-		err := rv.SetMetadata(k, v)
+		err = rv.SetMetadata(k, v)
 		if err != nil {
 			return fmt.Errorf("failed to set metadata key %q, value %q on image: %w", k, v, err)
 		}
 	}
 
+	if rv.ClusterName != "" {
+		if err = rv.SetMetadata(k8s.ClusterNameKey, rv.ClusterName); err != nil {
```

Comment:
just put the statements and `err != nil` on different lines like is done almost everywhere else in this project